tests/app/main/helpers/validation/test_g8_declaration.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+
+from app.main.helpers.validation import get_validator, G8Validator
+from app.main import content_loader
+
+
+FULL_G8_SUBMISSION = {
+    "termsAndConditions": False,
+    "understandTool": True,
+    "distortingCompetition": False,
+    "skillsAndResources": True,
+    "significantOrPersistentDeficiencies": False,
+    "cyberEssentials": False,
+    "companyRegistrationNumber": "N/A",
+    "dunsNumber": "123456789",
+    "registeredVATNumber": "N/A",
+    "distortedCompetition": False,
+    "conspiracyCorruptionBribery": True,
+    "contactNameContractNotice": "Test G8 Supplier",
+    "contactEmailContractNotice": "contact@email.com",
+    "organisationSize": "micro",
+    "graveProfessionalMisconduct": False,
+    "GAAR": False,
+    "canProvideCloudServices": True,
+    "bankrupt": False,
+    "confidentialInformation": False,
+    "seriousMisrepresentation": False,
+    "licenceOrMemberRequiredDetails": "I have a licence",
+    "understandHowToAskQuestions": True,
+    "primaryContactEmail": "primary@email.com",
+    "tradingStatusOther": "Sole trader",
+    "mitigatingFactors": "It wasn't me",
+    "taxEvasion": False,
+    "primaryContact": "The boss",
+    "subcontracting": [
+        "as a prime contractor, using third parties (subcontractors) to provide all services",
+        "as part of a consortium or special purpose vehicle, using members only to provide all services"
+    ],
+    "tradingStatus": "other (please specify)",
+    "terrorism": True,
```

Comment:
Supplier: `"terrorism": True`

Me: :fearful:

Supplier: `"mitigatingFactors": "It wasn't me"`

Me: :smile: